.github/workflows/integration.yml

```diff
@@ -9,7 +9,7 @@ on:
       - update-nixpkgs-*
   pull_request:
 env:
-  GO_VERSION: '1.22'
+  GO_VERSION: '1.21'
```

Comment:
I think you can still keep the Go version as 1.22 and build runc with 1.21.9 (https://github.com/cri-o/cri-o/pull/8011/files#diff-2864e5b0ec49a60b63556bdbf2ffa9c257ebae8656bb9d9ebbbf718d3eb40e8cR111) unless we decided to change the version to 1.21

Reply:
I thought runc main branch is failing to build with any go 1.22 right now

Reply:
Yeah, that's why we need to use the same hack as the one used in CI: https://github.com/cri-o/cri-o/pull/8011/files#diff-2864e5b0ec49a60b63556bdbf2ffa9c257ebae8656bb9d9ebbbf718d3eb40e8cR109-R129

Reply:
Reverted as suggested.